src/scripts/main.js

```diff
@@ -358,3 +358,18 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+const line = document.querySelector('tbody');
```

Comment:
Ensure that the 'tbody' element exists in the DOM before this script runs. You might want to wrap your code in a DOMContentLoaded event listener to ensure the DOM is fully loaded before executing this script.